gradle.properties

```diff
@@ -0,0 +1,67 @@
+modName = FTBLib
+
+# This is a case-sensitive string to identify your mod. Convention is to use lower case.
+modId = FTBL
+
+modGroup = latmod.lib
```

Comment:
Shouldn't this be ftb.lib? https://github.com/GTNewHorizons/FTB-Library/tree/master/src/main/java/ftb/lib

Reply:
was trying to stay as close to https://github.com/GTNewHorizons/FTB-Library/blob/e3019ae0cd62cf83f676284633e4da934fb59d18/build.gradle#L29 as possible without moving files around, perhaps ftb.lib is more correct? It has two folders.

Reply:
I guess LatvianModder interpreted it as maven group and not as path where the mod is